### ISSUE TITLE
Multiple Heatmap Components on Same Page. 

### ISSUE THREAD

**Issue by StephenMarkToms** [BrockReece/vue-heatmapjs#27]
Thank you for providing such a great library! I've been playing around with it but it looks like if I attach it to multiple elements that appear on the same page it breaks completely. 

for example this is the loop I'm using it on:

<pdf
            v-heatmap="showHeatmap"
            v-for="i in numPages"
            :key="i"
            :src="src"
            :page="i"
            class="w-full max-w-3xl mx-auto my-5"
        ></pdf>

The output shows the rendered pdfs but it does not highlight the mouse movement.

**Comment by BrockReece** [comment]
Hmm, in theory I'm not sure why it wouldn't work, but I don't think I have tried it in anger before, I will try to give it a test today.

Did you mean to include a code example above?

**Comment by StephenMarkToms** [comment]
@BrockReece Thank you for such a quick response. I must have missed adding the code snippet. This is what I meant to send earlier:

`
<template>
	<div class="relative w-full h-full">
         <pdf
            v-heatmap="showHeatmap"
            v-for="i in numPages"
            :key="i"
            :src="src"
            :page="i"
            class="w-full max-w-3xl mx-auto my-5"
        ></pdf>
	</div>
</template>

<script>

import pdf from 'vue-pdf'
import { pauser } from '../../app';

var loadingTask = pdf.createLoadingTask('http://localhost:3000/uploads/1597517593.pdf');

export default {
	components: {
		pdf
	},
	data() {
		return {
			src: loadingTask,
            numPages: undefined,
            paused: false,
            showScrollmap: false,
            showHeatmap: true,
		}
	},
    methods: {
        pauseCollection() {
            this.paused = !this.paused;
            pauser.next(this.paused);
            },
        },
        async mounted() {
            this.src.promise.then(pdf => {
                this.numPages = pdf.numPages;
            });
            window.addEventListener('load', () => {
            pauser.next(false);
            });
        },
    }

</script>

<style lang="scss" >
    .heatmap-canvas{
        border: solid 1px blue !important;
        width: 100% !important;
        height: 100% !important;
    }
</style>
`


Basically in my application I want to track interaction with a pdf and responsive div on top of another canvas rendered element to utilize the heatmap. If I remove the for loop and just display one pdf it will work as intended.

**Comment by BrockReece** [comment]
Yep, I have the same issue. I think I can see why though.

**Comment by BrockReece** [comment]
Can you give me a favour and test the changes in #28?
It is working for me locally.
![Aug-17-2020 15-01-11](https://user-images.githubusercontent.com/8988409/90405310-78d50300-e09b-11ea-9e24-6c60a494e250.gif)

**Comment by StephenMarkToms** [comment]
@BrockReece i will later this afternoon once I’m in front of a computer. Thank you so much for the prompt response.

**Comment by StephenMarkToms** [comment]
@BrockReece I can confirm this is now working and fixed in the latest commit. Thank you so much! Do you know if there's a way to make the heatmap responsive?

**Comment by StephenMarkToms** [comment]
I'll open a separate issue for this.
